seeds/dex/trades/dex_trades_seed.csv

```diff
@@ -340,4 +340,5 @@ arbitrum,kyberswap,elastic,2023-03-26,0x070e704ea6da35b05c97e2a2f9ccf01136f9bdf6
 bnb,sushiswap,1,2021-11-01 0:00,0xc66fb80242b7c788c2896b3dc8c611203771be9474d18f85cb2032d594de327c,830,0xbb4cdb9cbd36b01bd1cbaebf2de08d9173bc095c,0.07525109648,0x049d68029688eabf473097a2fc38ef61633a3c7a,43.351292
 bnb,sushiswap,1,2021-11-01 0:00,0xffc3e47949b600ee61d7b3b7b57da6137ed4c55652da396d02ff3ece60bfdd9b,591,0xbb4cdb9cbd36b01bd1cbaebf2de08d9173bc095c,0.01789196956,0x049d68029688eabf473097a2fc38ef61633a3c7a,9.541071
 polygon,sushiswap,1,2021-06-16 0:00,0xc625dc2c5e923afd8469e9e03233d37eab64d627a4e6f221c8b63efcf276523c,179,0x7ceb23fd6bc0add59e62ac25578270cff1b9f619,0.1131447356,0x0d500b1d8e8ef31e21c99d1db9a6444d3adf1270,191.734491
-polygon,sushiswap,1,2021-06-16 0:00,0xa727f3165a6e7533cbeecfbe67e4072f8b9a0eec486a4c9976b25b902a1bd6f0,310,0x7ceb23fd6bc0add59e62ac25578270cff1b9f619,0.0008197673253,0x0d500b1d8e8ef31e21c99d1db9a6444d3adf1270,1.388941988
\ No newline at end of file
+polygon,sushiswap,1,2021-06-16 0:00,0xa727f3165a6e7533cbeecfbe67e4072f8b9a0eec486a4c9976b25b902a1bd6f0,310,0x7ceb23fd6bc0add59e62ac25578270cff1b9f619,0.0008197673253,0x0d500b1d8e8ef31e21c99d1db9a6444d3adf1270,1.388941988
+avalanche_c,glacier,1,2023-04-21,0x8e5b51477a54c3de4e5850c09f33958ca357baa676bb7342389030bc4ba3e901,5,0x3712871408a829c5cd4e86da1f4ce727efcd28f6,60.778340365847253021,0xb31f66aa3c1e785363f0875a1b74e27b85fd66c7,0.1
```

Comment:
Also from the explorer, the token_bought_amount may be `63.768883740248004859`, but I am not very sure, please verify this part.

https://snowtrace.io/tx/0x8e5b51477a54c3de4e5850c09f33958ca357baa676bb7342389030bc4ba3e901

```suggestion
avalanche_c,glacier,1,2023-04-21,0x8e5b51477a54c3de4e5850c09f33958ca357baa676bb7342389030bc4ba3e901,5,0x3712871408a829c5cd4e86da1f4ce727efcd28f6,63.768883740248004859,0xb31f66aa3c1e785363f0875a1b74e27b85fd66c7,0.1
```

Reply:
https://dune.com/queries/2399354?d=1
And the block_date for this trade should be 2023-04-22 in UTC time.

Reply:
doh, thx